fenix/app/src/main/res/layout/custom_search_engine_radio_button.xml

```diff
@@ -1,39 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We're removing this because the only usage of this was from https://github.com/mozilla-mobile/firefox-android/pull/4055/files#diff-f156d151bcfa907ff18cdb71d644d8d334777451cfcfe6770b272ef14eae6b84L251